tmt/schemas/provision/hardware.yaml

```diff
@@ -354,6 +357,9 @@ definitions:
       virtualization:
         "$ref": "#/definitions/virtualization"
 
+      bootdisk:
```

Comment:
How about `boot-disk`?

Reply:
we can name it to boot-disk,but IMO bootdisk is better,to keep consistent with beaker value.But if you still think boot-disk is better,I can name it boot-disk to make you happy,so that this mr could be merged sooner:)

Reply:
we can name it to boot-disk,but IMO bootdisk is better,to keep consistent with beaker value.But if you still think boot-disk is better,I can name it boot-disk to make you happy,so that this mr could be merged sooner:)

Reply:
We try to consistently name multi-word keys with dashes, so `boot-disk` sounds like a better choice here. There are just a few exceptions but I think bootdisk is not so commonly used as for example `hostname`.